### PR TITLE
added some javascript to favorites.js file

### PR DESCRIPTION
**Pull request by Sarkissian321**

### CODE REVIEW

**Review by bellaloc — COMMENTED** [review]
hi meline, this looks good. here is the script that should be at the end of the code:

</ul>
<script src="./assets/js/favorites.js"></script>
</body>
</html>

otherwise it looks good. there was one other thing that might need to be defined which is the "lastSearch" on the "display favorites function"....  here - lastSearch[songIndex].track

File: assets/js/favorites.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+}
+
+function displayFavorites() {
+    var favoritesContainer = document.getElementById("favorites-list");
+    favoritesContainer.innerHTML = "";
+
+    if (favoritesList.length > 0) {
+        favoritesList.forEach(function (songIndex) {
+            var song = lastSearch[songIndex].track;
```
Comment (adamywfong) [inline assets/js/favorites.js:18]: lastSearch was defined in index.html but not favorites.html, so you might need to add localStorage for this item

File: assets/js/favorites.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            var listItem = document.createElement("li");
+            listItem.textContent = song.track_name + " by " + song.artist_name;
+            favoritesContainer.appendChild(listItem);
+        });
+    } else {
+        favoritesContainer.innerHTML = "You have no favorite songs yet.";
+    }
+}
+
+function addToFavorites(songIndex) {
```
Comment (adamywfong) [inline assets/js/favorites.js:28]: naming the parameter as songIndex seems to imply that is a number, but it looks like you're planning on storing song objects in here

**Review by adamywfong — APPROVED** [review]
Good start!

**Pull request merged**